modules/consentManagement.js

```diff
@@ -104,8 +104,6 @@ function lookupIabConsent(cmpSuccess, cmpError, hookConfig) {
         cmpSuccess(tcfData, hookConfig);
       } else if (tcfData.eventStatus === 'tcloaded' || tcfData.eventStatus === 'useractioncomplete') {
         cmpSuccess(tcfData, hookConfig);
```

Comment:
Just a question...but why not combine these two if/else if statements into a single one since the result is the same?

```javascript
if (tcfData.gdprApplies === false || tcfData.eventStatus === 'tcloaded' || tcfData.eventStatus === 'useractioncomplete') {
  cmpSuccess(tcfData, finalCallback);
}
```

Reply:
At this point, we can - I'll push an update.